ui/src/containers/AppDetailsToolbarContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { AppDetailToolbar } from '../components/AppDetailToolbar';
+
+export class AppDetailsToolbarContainer extends React.Component {
+  onDisableButton = () => {
+    console.log('Disable button clicked');
+  }
+
+  onHomeClick = () => {
+    this.props.history.push('/');
+  }
+
+  onSaveButton = () => {
```

Comment:
Could be named `onSaveApp` to be more descriptive. It is an event handler which could technically be used by something other that a button.